src/search.cpp

```diff
@@ -152,6 +152,19 @@ Search::Worker::Worker(SharedState&                    sharedState,
     clear();
 }
 
+// Function P interpolates parameter dependent on number of threads given values for one and eight threaded search.
+// For 8 threads exactly v8 is returned but because of rounding for one thread a value near v1 is returned (at most one unit of).
```

Comment:
There is at least a typo in the last word (`of` → `off`).

Suggestion for the whole comment:
```suggestion
// Interpolates a parameter for the current number of threads based on values for a single thread and for eight threads.
// For eight threads exactly v8 is returned; for a single thread the result may differ from v1 by at most 1 due to rounding errors.
```